### DISCUSSION TITLE
Neovim development environment recommendation | New to Kotlin

### DISCUSSION THREAD

**Discussion by thepragmaticmero** [Droid-ify/client#754] [General]
I'm quite new to programming and development in general, I could make my way around neovim, its plugins, terminals. (I daily drive Arch BTW). I'm gathering information about Neovim development on android, Droid-ify is a pretty cool project that I've had installed for the entirety of 2024 now and I love material design/material you.

Do any of the contributors have a Neovim development environment config that you could share with me? I'm trying to avoid the use of Android Studio :p (for ArchBTW purposes... minimalism). Is that something possible? Kotlin has an LSP so...

Good work with the app :D 

Android studio flatpak disk space for reference:
![image](https://github.com/Droid-ify/client/assets/169491353/3647874f-b74d-4f5c-8e48-773a2852cf54)

**Comment by Iamlooker** [comment]
I am not aware if anyone I know uses neovim for Android Development, but I will look into this and get back. I used to use Arch BTW 😁

**Comment by Srinath10X** [comment]
This might help you 
https://andrewcourter.substack.com/p/kotlin-setup-in-neovim

**Comment by Iamlooker** [comment]
I am planning to create a community driven IDE for android development, will need some help there. It will most probably be independent from neovim and might specifically target android only, but nothing is planned yet
